src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -51,6 +51,71 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
         }
 
         /// <summary>
+        /// Construct the resource prefix used to look up the resource.
+        /// </summary>
+        /// <param name="typeInfo">The type of the Resource to be looked up.</param>
+        /// <returns>A <see cref="string"/> representing the Resource name as it is stored without culture.</returns>
+        protected virtual string GetResourcePrefix(TypeInfo typeInfo)
+        {
+            if (typeInfo == null)
+            {
+                throw new ArgumentNullException(nameof(typeInfo));
+            }
+
+            return GetResourcePrefix(typeInfo, _hostingEnvironment.ApplicationName, _resourcesRelativePath);
+        }
+
+        /// <summary>
+        /// Construct the resource prefix used to look up the resource.
+        /// </summary>
+        /// <param name="typeInfo">The type of the Resource to be looked up.</param>
+        /// <param name="baseNamespace">The base namespace of the application.</param>
+        /// <param name="resourcesRelativePath">The folder containing all resources.</param>
+        /// <remarks>
+        /// For the type "Sample.Controllers.Home" if there's a resourceRelativePath return
+        /// "Sample.Resourcepath.Controllers.Home" if there isn't one then it would return "Sample.Controllers.Home".
+        /// </remarks>
+        /// <returns>A <see cref="string"/> representing the Resource name as it is stored without culture.</returns>
```

Comment:
More of a preference, `remarks` usually go at the end of the doc summary. Also, same issue as the previous returns and applies to the next method